src/components/pages/Theme/Theme.jsx

```diff
@@ -46,6 +49,209 @@ export const Theme = () => {
   const [isModalOpen, setIsModalOpen] = React.useState(false);
 
   // vars
```

Comment:
Can we move some of this variables to `constants` file, maybe? To keep this file smaller, for components only

Reply:
Actually, it's ok. We'll have a ticket for this. Once you merge this request, I can do a rebase and move it to its own file.